src/index.html

```diff
@@ -8,9 +8,37 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
+    <link rel="fonts" href="fonts/Avenir.ttc">
+    <link rel="fonts" href="fonts/Avenir-Book.ttf">
+    <link rel="fonts" href="fonts/Avenir-Heavy.ttf">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search">
+      <form class="search__panel search__panel--big" data-qa="big">
+        <img
+          src="images/Search.svg"
+          alt="search-icon"
+          class="search__icon search__icon--big"
+        >
+        <input
```

Comment:
Only a small area is clickable
Try to make the entire panel input (i.e clickable and display typed text)

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/186f6c10-f6bd-441c-8beb-10c3c78feb67)

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/4a0df451-92d6-4351-96a7-dccb313c7a75)

Also, on focus, there should be a gradient (check design) instead of the default `outline`